crates/bevy_render/src/pipelined_rendering.rs

```diff
@@ -83,13 +83,15 @@ impl Drop for RenderAppChannels {
 /// A single frame of execution looks something like below    
 ///
 /// ```text
-/// |--------------------------------------------------------------------|
-/// |         | RenderExtractApp schedule | winit events | main schedule |
-/// | extract |----------------------------------------------------------|
-/// |         | extract commands | rendering schedule                    |
-/// |--------------------------------------------------------------------|
+/// |---------------------------------------------------------------------------|
+/// |      |         | RenderExtractApp schedule | winit events | main schedule |
+/// | sync | extract |----------------------------------------------------------|
+/// |      |         | extract commands | rendering schedule                    |
+/// |---------------------------------------------------------------------------|
 /// ```
 ///
+/// - `sync` is the step where the entity-entity mapping between the main and render world is updated.
+///     This is run on the main app's thread, for more information checkout [`WorldSyncPlugin`].
```

Comment:
```suggestion
/// - `sync` is the step where the entity-entity mapping between the main and render world is updated. For more information checkout [`WorldSyncPlugin`].
///     This is run on the main app's thread, .
```
Or something to this effect. Just slightly group the sentences better (hard to figure out what it will look like on github, but I hope you get my point)